tests/acceptance/features/apiSpaces/copySpaces.feature

```diff
@@ -249,8 +249,8 @@ Feature: copy file
     And user "Alice" has accepted share "/testshare2" offered by user "Brian"
     When user "Alice" copies file "/testshare1/testshare1.txt" from space "Shares Jail" to "/testshare2/testshare1.txt" inside space "Shares Jail" using the WebDAV API
     Then the HTTP status code should be "201"
-    And for user "Alice" the space "Shares Jail" should contain these entries:
-      | /testshare2/testshare1.txt |
+    And for user "Alice" folder "testshare2" of the space "Shares Jail" should contain these files:
+      | testshare1.txt |
```

Comment:
what is the difference between those two steps?

Reply:
@individual-it inside `Shares jail` we have different shared resources which have their own id. And with specific resource we can do a `PROPFIND` request (with infinity depth) AFAIK. But `PROPFIND` inside `Shares jail` only support depth 0. https://github.com/owncloud/ocis/issues/4188. So for not relying to the issue to be fixed this step is used instead.